src/main/java/de/muenchen/dave/domain/Kalendertag.java

```diff
@@ -0,0 +1,39 @@
+package de.muenchen.dave.domain;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.LocalDate;
+
+@Entity
+@Getter
+@Setter
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class Kalendertag extends BaseEntity {
+
+    @Column(name = "datum", nullable = false, unique = true)
+    @JsonDeserialize(using = LocalDateDeserializer.class)
+    @JsonSerialize(using = LocalDateSerializer.class)
+    private LocalDate datum;
+
+    @Column(name = "ferientyp")
```

Comment:
Die Benamsung wird in den Annotationen nicht benötigt, da der Attributname übernommen wird.

Reply:
done